generators/client/templates/react/src/main/webapp/app/modules/login/login-redirect.tsx.ejs

```diff
@@ -0,0 +1,9 @@
+import React, { useEffect } from 'react';
+
+export const LoginRedirect = () => {
+  useEffect(() => window.location.reload());
+
+  return (<></>);
```

Comment:
Quick question, why not just return null?

Reply:
you're right, it will be simpler, I update